cmd/auth-provider-gcp/app/getcredentials.go

```diff
@@ -96,9 +102,9 @@ func defineFlags(credCmd *cobra.Command) {
 	credCmd.Flags().StringVarP(&authFlow, "authFlow", "a", gcrAuthFlow, "authentication flow (valid values are gcr, dockercfg, and dockercfg-url)")
```

Comment:
Please define a structure to hold configuration. Instantiate it as a local variable in NewGetCredentialsCommand() and pass it down to whatever needs access to the configuration.

Reply:
Done